src/hotspot/share/oops/stackChunkOop.inline.hpp

```diff
@@ -131,8 +131,10 @@ inline bool stackChunkOopDesc::is_in_chunk(void* p) const {
 }
 
 bool stackChunkOopDesc::is_usable_in_chunk(void* p) const {
-#if (defined(X86) || defined(AARCH64) || defined(RISCV64)) && !defined(ZERO)
+#if (defined(X86) || defined(AARCH64)) && !defined(ZERO)
   HeapWord* start = (HeapWord*)start_address() + sp() - frame::sender_sp_offset;
+#elif defined(RISCV64)
```

Comment:
I think for riscv64, we also need to exclude zero here? I mean: #elif defined(RISCV64) && !defined(ZERO)